### PR TITLE
Added Abhishikth.json file

### PR DESCRIPTION
**Pull request by Abhishikth3330**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1420"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thank you for contributing!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Comment by Abhishikth3330** [comment]
> Thank you for contributing!
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 👌 This code looks good  |  **Tags:** Maintainable

When will this profile be seen on LinkFree Site ?

File: public/data/Abhishikth.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        }
+    ],
+    "milestones": [
+        {
+            "title": "Bachelor's in Artificial Intelligence",
+            "date": "June 2019",
+            "color": "blue"
+        }
+    ]
+}
```
Comment (eddiejaoude) [inline public/data/Abhishikth.json:30]: Please add ending blank line as suggested by github

**Comment by Abhishikth3330** [comment]
> 

Eddie You are Amazinggggg !
My journey with Git & Github has started recently and you are the Motivation.
Thanks 😄

File: public/data/Abhishikth.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "avatar": "https://github.com/Abhishikth3330.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/Abhishikth3330",
+            "icon": "github"
+        },
+        {
+            "name": "Connect with me on LinkedIn",
+            "url": "",
```
Comment (eddiejaoude) [inline public/data/Abhishikth.json:14]: If you do not have a linkedin URL, you can remove this object

File: public/data/Abhishikth.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            "icon": "linkedin"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "",
```
Comment (eddiejaoude) [inline public/data/Abhishikth.json:19]: If you do not have a Twitter URL, you can remove this object

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Missing URLs for Twitter and LinkedIn socials

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**